.github/config/out_of_tree_extensions.cmake

```diff
@@ -81,5 +82,6 @@ if (NOT WIN32)
             LOAD_TESTS DONT_LINK
             GIT_URL https://github.com/duckdblabs/substrait
             GIT_TAG 5d621b1d7d16fe86f8b1930870c8e6bf05bcb92a
+            APPLY_PATCHES
```

Comment:
These changes (both substrait and sqlite_scanner) do makes sense also in isolation, so I get the patches but I would fine them independently (and ideally removing the patches before landing the PR or just afterward).

Reply:
Hmm fair they should just be upstreamed directly
I was more concerned with getting my CI to pass but you're right they should be separately pushed!

Reply:
PRs made:
<https://github.com/duckdb/substrait/pull/67>
<https://github.com/duckdb/sqlite_scanner/pull/71>